test_framework/terraform/aws/centos/variables.tf

```diff
@@ -94,10 +94,20 @@ variable "rke_k8s_version" {
   description = "RKE k8s version will be used to generate RKE config file output in case of arch=amd64"
 }
 
-variable "k3s_version" {
+variable "k8s_distro_name" {
+  type        = string
+  default     = "k3s"
+  description = "kubernetes distro version to install [rke, k3s]  (default: k3s)"
+}
+
+variable "k8s_distro_version" {
   type        = string
   default     = "v1.20.8+k3s1"
```

Comment:
We should update to the rather recent version later.

Reply:
fixed